kubernetes-support/README.md

```diff
@@ -0,0 +1,220 @@
+# Introduction
+The design for dspot on Kubernetes(K8s) is meant for deploying in parallel with repairnator as the initial goal(Check out this [issue](https://github.com/Spirals-Team/repairnator/issues/813) for more detais), but it can also be deployed completely alone without any side effects as design-wise it is identical to repairnator. 
```

Comment:
Here there is a mix between Repairnator and DSpot. This README should be completely independent of Repairnator, it should not mention it.

Reply:
Hi @monperrus , Sorry about that , it is fixed :) .